Student-Service/src/main/java/tw/waterball/judgegirl/studentservice/domain/usecases/SignUpUseCase.java

```diff
@@ -33,11 +33,11 @@
     private final StudentRepository studentRepository;
     private final PasswordEncoder passwordEncoder;
 
-    public void execute(Request request, Presenter presenter) {
+    public void execute(Request request, boolean isAdmin, Presenter presenter) {
```

Comment:
put `isAdmin` as a Request's attribute

all usecases should have the same method signature `void execute(Request request, Presenter presenter)`